server/test-chunkByOCR.ts

```diff
@@ -0,0 +1,263 @@
+import { promises as fsPromises } from "fs"
```

Comment:
do we need to commit these files?
we should add it in `test/` folder in that case